docs/known-limitations.rst

```diff
@@ -17,7 +17,8 @@ Example:
 
     import pytest
 
-    @pytest.mark.parametrize("param", {"a","b"})
+
+    @pytest.mark.parametrize("param", {"a", "b"})
```

Comment:
Misaligned space i just noted

Reply:
Yeah it was fixed when I ran `pre-commit run -a` locally.